src/system/file_path.cc

```diff
@@ -57,45 +57,43 @@ FilePath::FilePath(string file_path, const  FileType ft)
 
 
 
-void FilePath::set_io_dirs(const string working_dir, const string root_input_dir,const string input,const string output) {
-    // root directory
-    root_dir = root_input_dir;
-
+string FilePath::set_io_dirs(const string main_input_file, const string input, const string output, const string working_dir) {
     // relative output dir is relative to working directory
     // this is possibly independent of position of the main input file
```

Comment:
Yeah, bad decision in the past. Taking known use cases I see no reason to interpret relative output dirs differently then input dirs. This only leads to confusion.